tests/unittests/tests-gnrc_ipv6_nib/tests-gnrc_ipv6_nib-nc.c

```diff
@@ -245,9 +245,11 @@ static void test_nib_nc_mark_reachable__unmanaged(void)
     void *iter_state = NULL;
     static const ipv6_addr_t addr = { .u64 = { { .u8 = GLOBAL_PREFIX },
                                              { .u64 = TEST_UINT64 } } };
+    uint8_t l2addr[] = L2ADDR;
```

Comment:
`static const`?

Reply:
copy-pasta from other test-case above - could fix, but then in multiple places

Reply:
Since the topic of this PR is "fix gnrc_ipv6_nib test" I have no problem with that ;-)

Reply:
ah no, though its copy-pasta, in other tests `l2addr` is modified (i.e. `l2addr++`), so here `static const` but other places are good as is ...

Reply:
Wait... now that I think about it: why did you add the `l2addr` in the first place? `nc_set()` should work with `l2addr` being `NULL` (since there are link-layers like e.g. SLIP that don't have link-layer addresses).

Reply:
because of this [assert](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/network_layer/ipv6/nib/nib_nc.c#L32)

Reply:
Ohh... See https://github.com/RIOT-OS/RIOT/pull/8179 for fixing that.